drivers/input/Kconfig

```diff
@@ -45,13 +45,58 @@ config INPUT_TOUCHSCREEN_NPOLLWAITERS
 
 endif # INPUT_TOUCHSCREEN
 
+config INPUT_KEYBOARD
+	bool
```

Comment:
```suggestion
	bool "Enable keyboard"
```

Reply:
This configuration follows INPUT_TOUCHSCREEN.
When using UINPUT_KEYBOARD, it will be selected.

Reply:
ok.